libs/rapidsandrivers-extras/src/main/kotlin/rapidsandrivers/IkkeStandardiserteKeys.kt

```diff
@@ -20,6 +20,10 @@ const val GRUNNLAG_OPPDATERT = "grunnlag_oppdatert"
 const val OPPLYSNING_KEY = "opplysning"
 const val FNR_KEY = "fnr"
 
+const val OMREGNING_NYE_REGLER = "OMREGNING_NYE_REGLER"
```

Comment:
Kunne gjerne hatt eit eiget objekt for omregning her (tilsvarande feks ReguleringEvents), har på sikt lyst til å få bukt med både denne og StandardKeys, eller iallfall få tynna dei ut såpass at vi kan gi dei presise namn